pkg/nsx/services/common/store.go

```diff
@@ -85,3 +82,42 @@ func TransError(err error) error {
 	}
 	return err
 }
+
+// InitializeResourceStore is the method to query all the various resources from nsx-t side,
+// we could use it to cache all the resources in the store when process starts.
+func (service *Service) InitializeResourceStore(wg *sync.WaitGroup, fatalErrors chan error, resourceTypeValue string, store Store) {
```

Comment:
how about 
// InitializeResourceStore is the method to query all the various resources from nsx-t side and save them
to the store. 
// we could use it to cache all the resources in the store when process starts.

Reply:
ok